docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -77,6 +77,65 @@ At startup, this connector tries guessing fields' types, but it might not be cor
 
 This property is optional; the default is ``_schema``.
 
+.. note::
+
+    Trino can autogenerate the ``_schema`` contents by running a ``SELECT *`` over the collection.
+    However, not all the fields are always detected and you might need to update the schema manually or using ``ALTER TABLE``.
+
+
+Nested fields
+^^^^^^^^^^^^^
+
+Given a document with the following structure:
+
+.. code-block:: text
+
+    {
+      _id: <value>
+      nesting_1: {
+        nested_field_1_1: <value>
+        nested_field_1_2: <value>
+        ...
+      }
+      nesting_2: {
+        nesting_2_1: {
+          nested_field_2_1_1: <value>
+          nested_field_2_1_2: <value>
+          ...
+        }
+        nesting_2_2: {
+          nested_field_2_2_1: <value>
+          nested_field_2_2_2: <value>
+          ...
+        }
+      }
+    }
+
+You can define the nesting by using ``row`` as follows:
```

Comment:
Is this self-describing in the same document or in another file? Might want to be a bit more explicit.

Reply:
It's actually an entry on the MongoDB schema collection.

Reply:
Great. Should be explicit about that for our users. That's important for them to know.

Reply:
I updated it to be more explicit about the change.